app/build.gradle

```diff
@@ -41,6 +39,9 @@ dependencies {
     debugCompile 'com.squareup.leakcanary:leakcanary-android:1.5.1'
     releaseCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.1'
     testCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.1'
+
+    compile "io.reactivex.rxjava2:rxkotlin:$rx_version"
+    compile "io.reactivex.rxjava2:rxandroid:$rx_android_version"
```

Comment:
Oh, I always guided myself by the recommendation in the [RxAndroid repo](https://github.com/ReactiveX/RxAndroid#binaries):
```
// Because RxAndroid releases are few and far between, it is recommended you also
// explicitly depend on RxJava's latest version for bug fixes and new features.
```

Never really tried not to follow that recommendation. Is it better not to?